distribution/docs/src/main/resources/_contents/_configuring/configuring-solr-data-directory-contents.adoc

```diff
@@ -0,0 +1,46 @@
+
+==== Configuring Solr Catalog Provider Data Directory
```

Comment:
I'll check to see if it's being done as part of https://codice.atlassian.net/browse/DDF-2628